sdk/schemaregistry/azure-data-schemaregistry-apacheavro/pom.xml

```diff
@@ -15,11 +15,11 @@
   </parent>
 
   <groupId>com.azure</groupId>
-  <artifactId>azure-data-schemaregistry-avro</artifactId>
-  <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-data-schemaregistry-avro;current} -->
+  <artifactId>azure-data-schemaregistry-apacheavro</artifactId>
+  <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-data-schemaregistry-apacheavro;current} -->
```

Comment:
Could we do a Maven redirect release at the old coordinates too?

Reply:
Good call. I'll go and do that.